packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "@material/elevation/mixins";
+@import "@material/rtl/mixins";
+@import "@material/theme/mixins";
+@import "@material/typography/mixins";
+
+$mdc-toolbar-height: 64px;
+$mdc-toolbar-mobile-height: 56px;
+$mdc-toolbar-padding: 24px 16px;
+$mdc-toolbar-mobile-padding: 16px;
+
+.mdc-toolbar {
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: $mdc-toolbar-height;
+  padding: $mdc-toolbar-padding;
+  box-sizing: border-box;
+
+  @media (max-width: 599px) {
+    height: $mdc-toolbar-mobile-height;
+    padding: $mdc-toolbar-mobile-padding;
+  }
+
+  .mdc-toolbar__section {
+    display: inline-flex;
+    flex: 1;
+    align-items: baseline;
+    justify-content: center;
+
+    &.mdc-toolbar__section--align-start {
+      justify-content: flex-start;
+      order: -1;
+
+      @include mdc-rtl {
+        justify-content: flex-end;
+        order: 1;
+      }
+    }
+
+    &.mdc-toolbar__section--align-end {
+      justify-content: flex-end;
+      order: 1;
+
+      @include mdc-rtl {
+        justify-content: flex-start;
+        order: -1;
+      }
+    }
+
+    .mdc-toolbar__title {
+      @include mdc-typography(title);
+
+      order: 0;
```

Comment:
Good catch, `order` should be accidentally added. I already removed it.

`margin` is needed because `mdc-typography(title)` comes with a -0.5rem margin on the top. It is needed in other cases where title is used paragraph as title. However, in this case, it is more of a style rather than functioning as a title, so that minus margin pushed it off and needed to be removed.